auto/src/glew_head.c

```diff
@@ -126,7 +126,17 @@ void* NSGLGetProcAddress (const GLubyte *name)
 /*
  * Define glewGetProcAddress.
  */
-#if defined(GLEW_REGAL)
+#if defined(GLEW_CUSTOM_GET_PROC)
+#  ifndef GLEW_STATIC
+#  error GLEW_CUSTOM_GET_PROC currently requires GLEW_STATIC and defining the symbol GLEW_customGetProcAddress
```

Comment:
Perhaps `glewGetProcAddress` just ought to be used as-is, if already defined to something.
(A thought, not a firm opinion)

Reply:
Yeah sounds reasonable